app/views/layouts/application.html.slim

```diff
@@ -21,3 +21,5 @@ html
                 i.fa.fa-sign-out
     .container
       = yield
+    div id="footer"
```

Comment:
I think a better option would be to just get rid of this line completely - and just render the partial and make https://github.com/vpatel90/food-budgeter/pull/17/files#diff-c5481492b6a2dd94dbcf7604a99e2e7bR1 into `#footer` instead of `.footer`

that way all footer related code is in the partial